src/partials/table.html

```diff
@@ -1,32 +1,69 @@
-<section class="table">
-  <h2 class="table__header">Contact</h2>
-  <ul class="table__list">
-    <li class="table__box">
-      <h4 class="table__head">CAFE</h4>
-      <h5 class="table__place">Chicago</h5>
-      <p class="teble__content">Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
-      <div>Monady - Friday</div>
-      <div>06:00 AM - 10:00 PM</div>
-      <div>Saturday - Sunday</div>
-      <div>08:00 AM - 16:00 PM</div>
-    </li>
-    <li class="table__box">
-      <h4>FOOD TRUCK</h4>
-      <h5>Los Angeles</h5>
-      <p>Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
-      <div>Monady - Friday</div>
-      <div>06:00 AM - 10:00 PM</div>
-      <div>Saturday - Sunday</div>
-      <div>08:00 AM - 16:00 PM</div>
-    </li>
-    <li class="table__box">
-      <h4>CAFE</h4>
-      <h5>New York</h5>
-      <p>Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
-      <div>Monady - Friday</div>
-      <div>06:00 AM - 10:00 PM</div>
-      <div>Saturday - Sunday</div>
-      <div>08:00 AM - 16:00 PM</div>
-    </li>
-  </ul>
+<link href="../sass/main.css" rel="stylesheet" />
+<section class="table section">
+  <div class="background"></div>
+  <div class="container">
```

Comment:
Jeżeli to ma być użyta klasa globalna, którą ustaliliśmy to jej nazwa jest
section__container